common/networkpolicies/base/model-registry.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: model-registry
+  namespace: kubeflow
+spec:
+  podSelector:
+    matchExpressions:
+      - key: component
+        operator: In
+        values:
+          - model-registry-server
```

Comment:
Are you sure that this targets exactly/only the pods which serve on those ports? It should not target any model-registry pods that do not serve on those ports.

Reply:
Yes, this can be verified by showing that for the MR manifests, only MR is targeted, the other Deployments part of a MR installation do not have this selector, as also shown in this screenshot:

![image](https://github.com/kubeflow/manifests/assets/1699252/ff3ef02f-4cd4-4434-83c8-c0068166d695)

Reply:
I would consider this point resolved accordingly.